flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/task/context/StatefulTaskContext.java

```diff
@@ -184,6 +185,51 @@ private MySqlOffsetContext loadStartingOffsetState(
     }
 
     private boolean isBinlogAvailable(MySqlOffsetContext offset) {
+        String gtidStr = offset.gtidSet();
+        if (gtidStr != null) {
+            if (gtidStr.trim().isEmpty()) {
```

Comment:
Nit: Is it nesscessay to do ```trim()```  for the  ```gtidStr```?

Reply:
yes, debezium use sql command("SHOW MASTER STATUS")to get gtid set, it may be null, blank, or contain a GTID set.
![image](https://user-images.githubusercontent.com/44798441/148922140-d7024678-34f9-44a7-82f3-f96083a656f6.png)

Reply:
Ok, but the method ```trim()``` is try to remove the multiple blanks  for a string like ``` "     "```` -> ```""```.
So is it possible gtidStr that will contain multiple blanks? Or if you have ever  encountered such case?

Reply:
thanks, I did the verification and found nothing like this. I can add some comments and remove it?
![image](https://user-images.githubusercontent.com/44798441/150897266-f20eff67-7fb8-4872-bc60-6622cb99d84b.png)

Reply:
Thanks for the information. I just missed such code in debezium, and I think as such check also is done in debezium, we can also keep the `trim()`.